app/lua53/lnodemcu.c

```diff
@@ -427,7 +427,9 @@ static const char *readF (lua_State *L, void *ud, size_t *size) {
 
 static void eraseLFS(LFSflashState *F) {
   lu_int32 i;
+#ifdef LUA_USE_ESP
```

Comment:
Surely, the use of bare `printf()` statements for ESP code is generally deprecated.  We should use the appropriate equivalent.  Whatever we need a clear coding standard here.

Reply:
The `printf()`s aren't mine. I merely added the `#ifdef`s because having host-side LFS image generation tell me it's erasing flash was misleading at best.
(Okay, technically I believe I _moved_ one of the printfs to reduce the number of ifdefs I needed)

Reply:
Thinking about this, we should move towards POSIX compliant source wherever practical.   The real issue here is that printing should be deprecated, except for very specific circumstances, as it will be unusual for a human or logging system to be 'listening' to the serial output, mainly development debugging. This .... progress bar is really a development tool.